src/elements/content-sidebar/activity-feed/common/activity-message/ActivityMessage.scss

```diff
@@ -23,6 +23,27 @@
     color: $bdl-box-blue;
 }
 
+.bcs-ActivityMessage-truncated {
+    /* webkit-line-clamp only works in combinatio with -webkit-box */
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
+    max-height: 140px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    overflow-wrap: break-word;
+    -webkit-line-clamp: 7;
+    -webkit-box-orient: vertical;
```

Comment:
I'm not familiar with these or `display: -webkit-box`. what does this do? does this work in other browsers (Firefox, Safari)?

Reply:
It allows to define after how many lines text should be cut, and adds `...` at the end
It is standardized for legacy support

https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp
https://caniuse.com/css-line-clamp